### ISSUE TITLE
NoClassDefFoundError on InvalidCoordinatesException with Selenium 4.1.3

### ISSUE THREAD

**Issue by snicoll** [SeleniumHQ/htmlunit-driver#117]
Using the latest Selenium release (`4.1.3`) leads to:

```
Exception in thread "pool-3-thread-1" java.lang.NoClassDefFoundError: org/openqa/selenium/interactions/InvalidCoordinatesException
	at org.openqa.selenium.htmlunit.HtmlUnitDriver.resetKeyboardAndMouseState(HtmlUnitDriver.java:605)
	at org.openqa.selenium.htmlunit.HtmlUnitDriver.get(HtmlUnitDriver.java:600)
	at org.openqa.selenium.htmlunit.HtmlUnitDriver.lambda$get$7(HtmlUnitDriver.java:560)
	at org.openqa.selenium.htmlunit.HtmlUnitDriver.lambda$runAsync$0(HtmlUnitDriver.java:292)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.lang.ClassNotFoundException: org.openqa.selenium.interactions.InvalidCoordinatesException
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:581)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
```

See also https://github.com/SeleniumHQ/selenium/issues/10502

It looks like this class was removed in `4.1.3`.

**Comment by rbri** [comment]
Will try to fix this for the next Release (planned for this weekend).

BTW - you are aware or https://github.com/spring-projects/spring-framework/issues/28240

**Comment by rbri** [comment]
3.61.0-SNAPSHOT is available - please check

**Comment by rbri** [comment]
fixed in version 3.61.0
